docs/root/intro/arch_overview/upstream/load_balancing/load_balancers.rst

```diff
@@ -45,16 +45,19 @@ same or different weights.
 
   In this case the weights are calculated at the time a host is picked using the following formula:
 
-  `weight = load_balancing_weight / (active_requests + 1)^active_requests_exponent`.
+  `weight = load_balancing_weight / (active_requests + 1)^active_request_bias`.
 
-  :ref:`active_requests_exponent<envoy_v3_api_field_config.cluster.v3.Cluster.LeastRequestLbConfig.active_requests_exponent>`
-  can be configured via runtime and defaults to 1.0.
+  :ref:`active_request_bias<envoy_v3_api_field_config.cluster.v3.Cluster.LeastRequestLbConfig.active_request_bias>`
+  can be configured via runtime and defaults to 1.0. It must be greater than or equal to 0.0.
 
-  If `active_requests_exponent` is set to 0.0, the least request load balancer behaves like the
-  round robin load balancer and ignores the active request count at the time of picking.
+  The larger the active request bias is, the more aggressively active requests will lower the
+  effective weight.
 
-  For example, if active_requests_exponent is 1.0, a host with weight 2 and an active request count
-  of 4 will have an effective weight of 2 / (4 + 1)^1 = 0.4. This algorithm provides good balance at
+  If `active_request_bias` is set to 0.0, the least request load balancer behaves like the round
```

Comment:
Might want to make it clear that this only happens if weights are set for various endpoints.

Reply:
@tonya11en this whole section talks about what happens when weights are not equal (the section section starts with `*all weights not equal*:`).

Do you think we should still add a clarification or would it be redundant?

Reply:
oh, it's redundant